open-metadata-implementation/access-services/analytics-modeling/analytics-modeling-server/src/main/java/org/odpi/openmetadata/accessservices/analyticsmodeling/synchronization/ExecutionContext.java

```diff
@@ -193,5 +194,15 @@ public AuditLog getAuditLog() {
 		return auditLog;
 	}
 	
-	
+	/**
+	 * Helper function to get string property from property collection of the entity.
+	 * @param propName property name to retrieve.
+	 * @param properties collection.
+	 * @param methodName requested the operation.
+	 * @return property value.
+	 */
+	public String getStringProperty(String propName, InstanceProperties properties, String methodName)
```

Comment:
Why not use the repository helper directly?

Reply:
it is easier to use, the ExecutionContext provides all necessary details like helper and serviceName

Reply:
But this is a duplication of code that will then need to be fixed in two places.

Reply:
This means that we have multiple implementations of the same logic